### ISSUE TITLE
Some SVG and Path elements not correctly rendered in the browser with JPro

### ISSUE THREAD

**Issue by besidev** [JPro-one/JPro-tickets#182]
### Issue Summary:
When running a JavaFX application with JPro, some SVG shapes and Path elements are not rendered correctly in the browser. Specifically, the fill rules for paths (such as `EVEN_ODD` and `NON_ZERO`) do not behave as expected, resulting in incorrect visual representation of complex shapes.

### Expected Behavior:
The shapes should render in the browser exactly as they do when running the application locally. The `EVEN_ODD` fill rule should correctly fill or leave empty the inner areas of complex shapes.

### Actual Behavior:
The shapes do not render correctly in the browser. The inner areas that should remain unfilled when using the `EVEN_ODD` fill rule are incorrectly filled, or the entire path does not render as expected.

### Environment:
* JavaFX version: 17+
* JPro version: `2024.3.0`
* Browser: Any browser that fully supports SVG rendering with complex shapes and rules
* Operating System: Linux, Windows, macOS

### Example Code:
Very simple example applications reproducing the issue.

#### Using `Path` object:
```java
import javafx.application.Application;
import javafx.scene.Scene;
import javafx.scene.layout.StackPane;
import javafx.scene.paint.Color;
import javafx.scene.shape.ClosePath;
import javafx.scene.shape.FillRule;
import javafx.scene.shape.HLineTo;
import javafx.scene.shape.MoveTo;
import javafx.scene.shape.Path;
import javafx.scene.shape.VLineTo;
import javafx.stage.Stage;

public class PathExample extends Application {

    @Override
    public void start(Stage primaryStage) {
        // Create the Path
        Path path = new Path();
        
        // Set the fill rule to EVEN_ODD
        path.setFillRule(FillRule.EVEN_ODD);
        
        // Add elements corresponding to the SVG path commands
        path.getElements().addAll(
            new MoveTo(110, 0),         // M110,0
            new HLineTo(200),           // h90 (110+90)
            new VLineTo(90),            // v90 (0+90)
            new HLineTo(110),           // h-90 (200-90)
            new ClosePath(),            // z
            new MoveTo(130, 20),        // M130,20
            new HLineTo(180),           // h50 (130+50)
            new VLineTo(70),            // v50 (20+50)
            new HLineTo(130),           // h-50 (180-50)
            new ClosePath()             // z
        );
        
        // Set fill and stroke for the path
        path.setFill(Color.LIGHTBLUE);
        path.setStroke(Color.BLACK);
        
        // Create a StackPane to hold the path
        StackPane root = new StackPane();
        root.getChildren().add(path);
        
        // Create the scene and add it to the stage
        Scene scene = new Scene(root, 300, 200);
        primaryStage.setTitle("JavaFX Path Example");
        primaryStage.setScene(scene);
        primaryStage.show();
    }

    public static void main(String[] args) {
        launch(args);
    }
}
```

#### Using `SVGPath` object:
```java
import javafx.application.Application;
import javafx.scene.Scene;
import javafx.scene.layout.StackPane;
import javafx.scene.paint.Color;
import javafx.scene.shape.FillRule;
import javafx.scene.shape.SVGPath;
import javafx.stage.Stage;

public class SVGPathExample extends Application {

    @Override
    public void start(Stage primaryStage) {
        // Create the SVGPath
        SVGPath svgPath = new SVGPath();
        svgPath.setContent("M110,0 h90 v90 h-90 z M130,20 h50 v50 h-50 z");
        
        // Set the fill rule to EVEN_ODD
        svgPath.setFillRule(FillRule.EVEN_ODD);
        
        // Set fill and stroke for the SVGPath
        svgPath.setFill(Color.LIGHTBLUE);
        svgPath.setStroke(Color.BLACK);
        
        // Create a StackPane to hold the SVGPath
        StackPane root = new StackPane();
        root.getChildren().add(svgPath);
        
        // Create the scene and add it to the stage
        Scene scene = new Scene(root, 300, 200);
        primaryStage.setTitle("SVGPath Example");
        primaryStage.setScene(scene);
        primaryStage.show();
    }

    public static void main(String[] args) {
        launch(args);
    }
}
```

In both cases, when the applications are running in the browser via JPro we get the following result:
<img width="128" alt="fill-rule-incorrect" src="https://github.com/user-attachments/assets/ce5dfda7-4e7e-4c5a-8b16-99d0262fdbac">
While, when running the same apps locally, we get the correct and expected result:
<img width="128" alt="fill-rule-correct" src="https://github.com/user-attachments/assets/18220a13-bdd8-4482-ac9c-2b182d67940a">

### Proposed Solution:
This issue might be related to how JPro handles the rendering of JavaFX’s Path and SVG elements in the browser. Investigate the handling of SVG and Path elements in JPro and ensure that the fill rules and rendering logic align with standard JavaFX behavior.

**Comment by besidev** [comment]
Another example of incorrect rendering of a complex SVG icon (outlined with the red rectangle):

<img width="280" alt="complex-svgicon-incorrect-render" src="https://github.com/user-attachments/assets/518573c2-20e6-4c1d-ad41-649911619533">

**Comment by besidev** [comment]
The current fix ensures that the fill rules (such as `EVEN_ODD` and `NON_ZERO`) are properly applied and rendered in accordance with JavaFX’s standard behavior.

Considering the previous example, here is the rendering of the second icon (from the left) after the fix:

<img width="280" alt="complex-svgicon-correct-render" src="https://github.com/user-attachments/assets/c9807420-10f9-44af-88d6-f26eb2e1c5c1">

**Comment by besidev** [comment]
Here is the combined result of all examples rendering correctly with the given fill rules for both `Path` and `SVGPath` objects:

1. The first column involves a test with different fill rules using Path objects.
2. The second column involves a test with different fill rules using SVGPath.
3. The third column involves a test on rendering a complex SVG icon.

<img width="488" alt="fill-rules-correct-rendering" src="https://github.com/user-attachments/assets/f5a6ed08-ce01-4aa0-918e-e954c7780257">

**Comment by besidev** [comment]
The issue was fixed in **JPro** version `2024.3.1`
